wavelink/ext/spotify/__init__.py

```diff
@@ -443,7 +443,7 @@ async def _search(self,
             data = await resp.json()
 
         if data['type'] == 'track':
-            return SpotifyTrack(data)
+            return [SpotifyTrack(data)]
```

Comment:
This is a breaking change for people who are not using return_first

Reply:
I propose this because return_first is currently deprecated,  as it is only used when dealing with SpotifySearchType.track, but node._spotify._search method on line 282 will always return a single SpotifyTrack object, so when the client use return_first parameter, SpotifyTrack's search method will try to get the first item from tracks variable but it'll throw an IndexError since it's not a list. This kind of error it's tricky to catch 'cause it's not raised in the terminal. I spent some time figuring this out.

Reply:
But you're right, it's best just disabled this return_first flag for now. This will prevent the IndexError be raised when using this flag and will return exactly the same thing as before